tests/phpunit/admin/class-wc-stripe-whats-new-modal-test.php

```
@@ -0,0 +1,225 @@
+
+		$this->assertIsArray( $entries );
+
+		// The bundled readme.txt may or may not include the constant version
+		// (the changelog is updated per release). When it does, every entry
+		// must carry text; a tag is optional.
+		foreach ( $entries as $entry ) {
+			$this->assertArrayHasKey( 'tag', $entry );
+			$this->assertArrayHasKey( 'text', $entry );
+			$this->assertNotSame( '', $entry['text'] );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_parse_changelog_returns_entries_for_current_version is not named in camelCase. (undefined)

(CamelCaseMethodName)